docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -1,66 +1,129 @@
 # Configurations for Intel® Processor Graphics (GPU) with OpenVINO™ {#openvino_docs_install_guides_configurations_for_intel_gpu}
 
-
 @sphinxdirective
 
 .. _gpu guide:
 
-
-To use the OpenVINO™ GPU plugin and offload inference to Intel® Processor Graphics (GPU), Intel® Graphics Driver must be properly configured on your system.
-
-If Intel® Graphics Driver is already installed and you would like to keep it, you can skip the installation steps below.
+To use the OpenVINO™ GPU plug-in and transfer the inference to the graphics of the Intel® processor (GPU), the Intel® graphics driver must be properly configured on the system.
 
 Linux
 #####
 
-To install the latest available **Intel® Graphics Compute Runtime for oneAPI Level Zero and OpenCL™ Driver** for your operating system, 
-see its `installation guide on GitHub <https://github.com/intel/compute-runtime/releases/latest>`_.
+To use a GPU device for OpenVINO inference, you must meet the following prerequisites:
 
-.. note::
+- use a supported Linux kernel as per the `documentation <https://dgpu-docs.intel.com/driver/kernel-driver-types.html>`__
+- install GPU Runtime drivers: `The Intel(R) Graphics Compute Runtime for oneAPI Level Zero and OpenCL(TM) Driver <https://github.com/intel/compute-runtime/releases/latest>`__
+- install `Intel Graphics Memory Management Library <https://github.com/intel/gmmlib>`__ and `Intel® Graphics Compiler for OpenCL™ <https://github.com/intel/intel-graphics-compiler>`__
+- install `OpenCL ICD loader package <https://github.com/KhronosGroup/OpenCL-ICD-Loader>`__
+
+Depending on your operating system, there may be different methods to install the above packages. Below are the instructions on how to install the packages on supported Linux distributions.
 
-  If you are using RedHat 8, you can install the OpenCL library as a prerequisite by using the following command: 
-  ``http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/ocl-icd-2.2.12-1.el8.x86_64.rpm``
+Ubuntu 22.04 LTS
+++++++++++++++++
 
+Download and install the `deb` packages published `here <https://github.com/intel/compute-runtime/releases/latest>`__ and install the apt package `ocl-icd-libopencl1` with the OpenCl ICD loader.
 
+Alternatively, add the apt repository (as described here `here <https://dgpu-docs.intel.com/driver/installation.html>`__). For example, for an ARC device, use the command:
 
-You may consider installing one of the earlier versions of the driver, based on your particular setup needs.
+.. code-block:: sh
+   
+   apt-get update && apt-get install -y gpg gpg-agent curl
+   curl https://repositories.intel.com/graphics/intel-graphics.key | \
+     gpg --dearmor --output /usr/share/keyrings/intel-graphics.gpg
+   echo 'deb [arch=amd64,i386 signed-by=/usr/share/keyrings/intel-graphics.gpg] https://repositories.intel.com/graphics/ubuntu jammy arc' | \
```

Comment:
installation of GPU drivers is external thing with regards to OpenVINO.
We don't want to show exact packages which need to be installed. Supposed that users should configure GPU before running OpenVINO.

Reply:
@ilya-lavrenov relying purely on external GPU documentation has two problems: 
- the documentation include dependencies not needed by openvino like GUI applications dependencies and media decoding accelerators.
- there are packages which are not mentioned as required like ocl-icd
I think be adding a guide which is easy to follow and more specific for OV will save us and the users time on troubleshooting.

Reply:
`ocl-icd` is a direct dependency of Intel GPU and we have to add it to our `install_openvino_dependencies.sh` script.

> the documentation include dependencies not needed by openvino like GUI applications dependencies and media decoding accelerators.

This is not OpenVINO problem, we are not going to rewrite documentation of our dependencies.

Reply:
Added ICD loader as a dependency https://github.com/openvinotoolkit/openvino/pull/18123